### PR TITLE
feat (search) :

### PR DESCRIPTION
**Pull request by mmjedidi**
- add new feature search

### CODE REVIEW

File: app/src/main/java/com/dev0ko/ewelp/view/home/MainActivity.kt [hunk 1]
```diff
@@ -50,7 +56,20 @@ class MainActivity : AppCompatActivity() {
 
             binding.recommend.adapter = recommendadapter
         })
-
+        binding.searchField.setOnEditorActionListener { v, actionId, event ->
+            if (actionId == EditorInfo.IME_ACTION_SEND) {
+                val searchText = binding.searchField.text.toString()
+                Log.println(Log.ASSERT, "searched value", searchText)
```
Comment (kchaou-fakhri) [inline app/src/main/java/com/dev0ko/ewelp/view/home/MainActivity.kt]: Remove the log

File: app/src/main/java/com/dev0ko/ewelp/view/image/ListOfImagesActivity.kt [hunk 2]
```diff
@@ -72,6 +73,7 @@ class ListOfImagesActivity : AppCompatActivity() {
                 or View.SYSTEM_UI_FLAG_HIDE_NAVIGATION)
     }
     private fun getData(category: String?, page: Int) {
+        Log.println(Log.ASSERT,"category",category.toString())
```
Comment (kchaou-fakhri) [inline app/src/main/java/com/dev0ko/ewelp/view/image/ListOfImagesActivity.kt]: Remove the Log

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -85,6 +85,8 @@
             android:drawableLeft="@drawable/ic_round_search"
             android:drawablePadding="10dp"
             android:hint="Find wallpaper..."
+            android:imeOptions="actionSend"
```
Comment (kchaou-fakhri) [inline app/src/main/res/layout/activity_main.xml]: I think w need to use actionSearch instead of actionSend, because our action is search

Comment (mmjedidi) [reply app/src/main/res/layout/activity_main.xml]: yeah i will change it

**Pull request merged**